lib/datadog/core/configuration/components.rb

```diff
@@ -81,13 +82,17 @@ def build_telemetry(settings, agent_settings, logger)
         def initialize(settings)
           @logger = self.class.build_logger(settings)
 
+          # This agent_settings is intended for use within Core. If you require
+          # agent_settings within a product outside of core you should extend
+          # the Core resolver from within your product/component's namespace.
```

Comment:
I ultimately decided to not create component-specific resolvers for telemetry and remote. It felt odd since they already live in core and it's not obvious to me they'd ever have specific agent settings to resolve beyond what is provided in core. Happy to add them if people disagree.

Reply:
I think that's reasonable. :+1: We can always add them later if we really need them.